<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>607</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>607</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_Code">
      <property name="text">
       <string>Шифр образца</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_SentToCustomer">
      <property name="text">
       <string>Отправлен заказчику</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_AccountNumber"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Company"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_Comment">
      <property name="text">
       <string>Комментарий к договору</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_DateReceiptResults">
      <property name="text">
       <string>Дата получения результатов</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QTextEdit" name="textEdit_Comment"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_DateOfReceipt">
      <property name="text">
       <string>Дата получения</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QCheckBox" name="checkBox_Urgent">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_AccountNumber">
      <property name="text">
       <string>Номер счёта</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_Code"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_Company">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Компания (заказчик)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QDateEdit" name="dateEdit_DatePay">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDateEdit" name="dateEdit_DateTransferLaboratory">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_DatePay">
      <property name="text">
       <string>Дата оплаты</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="dateEdit_DateOfReceipt">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_DateTransferLaboratory">
      <property name="text">
       <string>Дата передачи образца в лабораторию</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_Urgent">
      <property name="text">
       <string>Срочный</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QCheckBox" name="checkBox_SentToCustomer">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_ContractNumber"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_ContractNumber">
      <property name="text">
       <string>Номер договора</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDateEdit" name="dateEdit_DateReceiptResults">
      <property name="displayFormat">
       <string>dd.MM.yyyy</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_Sample">
      <property name="text">
       <string>Образец</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_Sample"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
